### PR TITLE
refactor: Lower camel case

### PR DESCRIPTION
**Pull request by mfederowicz**
### Summary

Cleanup some parts of code to meet one of the standards:

- lowerCamelCase names capitalize the first letter of each word, except the first which is always lowercase, even if it’s an acronym -  ([source](https://dart.dev/guides/language/effective-dart/style))

This PR is a squashed version of https://github.com/EXERLOG/exer_log/pull/169 as @tenshiAMD suggested to close previous and create new one (becuse of my lack of rebasing knowledge, especially in forked repos in github :()

### CODE REVIEW

**Review by tenshiAMD — CHANGES_REQUESTED** [review]
@mfederowicz hi I think we should first fix the line endings issue. I will do that later. Please wait until then. Thanks!

**Comment by mfederowicz** [comment]
As Arnie said "No problemo" :)

**Comment by tenshiAMD** [comment]
@mfederowicz hi first pull the latest changes in alpha branch of the main repo then kindly run these commands in your working branch

```
git fetch origin
git reset --hard origin/alpha
```

after that start working with your changes. Cheers! 🎉

**Comment by tenshiAMD** [comment]
@mfederowicz looks like you did not pull the latest updates in alpha of the main repo yet

**Comment by mfederowicz** [comment]
@tenshiAMD ok i get it
to sync you fork:
```
git checkout alpha
git remoute update
git merge upstream/alpha
git push
```
to sync latest changes from origin/alpha - this is my fork copy
```
git fetch origin
git reset --hard origin/alpha
```
after that i should git pull, and merge some changes arrives from alpha - thats ok (Android Studio have good merging tools)

but what about case when I create multiple changes in few files, and push commits to remote, some fixes, and changes came from PR, another commits, so what about that, How should I prepare branch with my changes, how to rebase/squash them? (I know how to rebase local commits before push, but Iam afraid of making changes in already pushed commits) Any good advise or clean tutorial?

**Comment by tenshiAMD** [comment]
@mfederowicz your PR should only have one or two commits from the alpha.

**Comment by mfederowicz** [comment]
> @mfederowicz your PR should only have one or two commits from the alpha.

Yes i know but i think git reset  - - hard
was not a good idea, now we have 10 commits and 66 files changed instad od 6 :(

**Comment by tenshiAMD** [comment]
@mfederowicz we should not see any merge commit too. I think you need to use cherry-pick after rebasing to alpha.

**Comment by tenshiAMD** [comment]
@mfederowicz sorry for being strict, please run these commands in your working branch to save you some time (I already cleanup up and created the same branch in my fork).

```
git remote add amd https://github.com/tenshiAMD/exer_log
git fetch --all
git reset --hard amd/lowerCamelCase
git push -f origin lowerCamelCase
```

I hope this helps you. Thanks!

File: lib/UI/exercise/set_widget.dart
Comment (tenshiAMD) [inline lib/UI/exercise/set_widget.dart]: @mfederowicz please remove this line

**Review by tenshiAMD — CHANGES_REQUESTED** [review]
@mfederowicz please check the requested changes. Thanks!

**Comment by mfederowicz** [comment]
@tenshiAMD ok it is very helpfull, but it means that you prepare branch, and I pushed it to my fork used --force switch, but is it possible to cleanup (sqash) all commits to one without creating new temporary branch?

File: lib/UI/exercise/set_widget.dart
Comment (mfederowicz) [reply lib/UI/exercise/set_widget.dart]: done

**Comment by tenshiAMD** [comment]
> @tenshiAMD ok it is very helpfull, but it means that you prepare branch, and I pushed it to my fork used --force switch, but is it possible to cleanup (sqash) all commits to one without creating new temporary branch?

@mfederowicz yeah use to do sorcery xD basically, you can do force push and commit fixup together to achieve this kind of cleanup.

**Comment by tenshiAMD** [comment]
@mfederowicz LGTM now. I think we can merge this after passing all tests. Cheers! 🎉

**Review by tenshiAMD — APPROVED** [review]

**Comment by mfederowicz** [comment]
> @mfederowicz LGTM now. I think we can merge this after passing all tests. Cheers! tada

@tenshiAMD sorry for my silly questions, but I have many years web experience, but github few months already (especialy collaboration in multi dev project). In my current projects I use gitlab, and dont use forks, so I must level up my github skills :)

**Comment by tenshiAMD** [comment]
> @tenshiAMD sorry for my silly questions, but I have many years web experience, but github few months already (especialy collaboration in multi dev project). In my current projects I use gitlab, and dont use forks, so I must level up my github skills :)

@mfederowicz yeah you can learn a lot especially when contributing or collaborating with other devs. Gitlab also uses git so they are basically the same.

**Comment by tenshiAMD** [comment]
@all-contributors please add @mfederowicz for code

**Pull request merged**
